src/Apps/Order/Routes/Review/index.tsx

```diff
@@ -66,12 +71,49 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
               },
             },
             onCompleted: result => {
-              if ("error" in result.submitOrder.orderOrError) {
-                this.onMutationError(result.submitOrder.orderOrError.error)
-                return
-              }
+              const {
+                submitOrder: { orderOrError },
+              } = result
+
+              const error = orderOrError.error
+              if (error) {
+                switch (error.code) {
+                  case "insufficient_inventory":
+                    const artistId = get(
+                      this.props.order,
+                      o => o.lineItems.edges[0].node.artwork.artist.id
```

Comment:
what happens when there are multiple artists? https://github.com/artsy/metaphysics/blob/master/src/schema/artwork/index.ts#L78

Reply:
We'll have to choose one anyway-- `artist` grabs the first artist when it's returned from gravity, but I can explicitly query for `artists` here and grab the first one client-side to be more explicit.